src/formatDate.js

```diff
@@ -50,7 +50,57 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const factDate = date.split(fromFormat[fromFormat.length - 1]);
+  const fromFormatCopy = [...fromFormat];
+  const resultArray = [...toFormat];
+
+  fromFormatCopy.splice(-1, 1);
+  resultArray.splice(-1, 1);
+
+  if ((fromFormatCopy.includes('YYYY') && toFormat.includes('YYYY'))
```

Comment:
```suggestion
const YEAR_LONG = 'YYYY';

  if ((fromFormatCopy.includes(YEAR_LONG) && toFormat.includes(YEAR_LONG))
```
Consider using variables with meaningful names instead of magic strings